src/styles/App.css

```diff
@@ -1,40 +1,21 @@
-.App {
-  text-align: center;
+* {
+  box-sizing: border-box;
+  outline: none;
+  font-family: 'Nunito Sans', sans-serif;
```

Comment:
The default font family for the application should be Lato. So you can update this.